cnf-certification-test/networking/declaredandlistening/declaredandlistening.go

```diff
@@ -43,10 +43,12 @@ func ParseListening(res string, listeningPorts map[Key]bool) {
 			return
 		}
 		s := strings.Split(fields[indexport], ":")
-		p, _ := strconv.Atoi(s[1])
+		p, _ := strconv.Atoi(s[len(s)-1])
```

Comment:
Check if len(s) is > 0 before using len(s)-1, otherwise print an error and continue

Reply:
I did it , but which error do you want that I will print?